src/choropleth.ts

```diff
@@ -856,12 +895,15 @@ function showMapCallout(data, event, d): void {
     d3.select("div#map-callout");
 
   // set the callout title text
-  callout.select("#map-callout-title").text(d.properties.name);
+  console.log(d)
```

Comment:
Should we remove all the console logs from the final code?

Reply:
removed.